frontend/pages/teams/[slug]/manage.tsx

```diff
@@ -1,70 +1,75 @@
 import Button from '@/components/atoms/Button'
 import Paginate from '@/components/organisms/Paginate'
 import Table, { ColumnType, DataType } from '@/components/organisms/Table'
-import { PaginatorInfo } from '@/pages/questions'
+import GET_MEMBERS from '@/helpers/graphql/queries/get_members'
+import { loadingScreenShow } from '@/helpers/loaderSpinnerHelper'
+import { errorNotify } from '@/helpers/toast'
+import { useQuery } from '@apollo/client'
 import Link from 'next/link'
-import React from 'react'
+import { useRouter } from 'next/router'
+
+const columns: ColumnType[] = [
+    {
+        title: 'Name',
+        key: 'name',
+    },
+    {
+        title: 'Role',
+        key: 'role',
+    },
+    {
+        title: 'Action',
+        key: 'action',
+        width: 20,
+    },
+]
+
+interface IMember {
+    id: number
+    teamRole: {
+        id: number
+        name: string
+    }
+    user: {
+        id: number
+        first_name: string
+        last_name: string
+    }
+}
 
 const TeamManage = () => {
     const teamName = 'Sun Overflow'
     const teamSlug = 'sun-overflow'
 
-    const pageInfo: PaginatorInfo = {
-        perPage: 10,
-        currentPage: 1,
-        lastPage: 3,
-        total: 20,
-        hasMorePages: true,
-    }
-
-    const onPageChange = () => {
-        console.log('Change page...')
-    }
+    const router = useRouter()
 
-    const columns: ColumnType[] = [
-        {
-            title: 'Name',
-            key: 'name',
-        },
-        {
-            title: 'Role',
-            key: 'role',
-        },
-        {
-            title: 'Action',
-            key: 'action',
-            width: 20,
+    const { data, loading, error, refetch } = useQuery(GET_MEMBERS, {
+        variables: {
+            teamSlug: router.query.slug,
+            first: 5,
+            page: 0,
         },
-    ]
+    })
 
-    const data: DataType[] = [
-        {
-            key: 1,
-            name: 'John Doe',
-            role: 'Team Leader',
-        },
-        {
-            key: 2,
-            name: 'Precious Gift',
-            role: 'Alice Gods',
-        },
-        {
-            key: 3,
-            name: 'Jhon Rhon',
-            role: 'Benedetta Gods',
-        },
-        {
-            key: 4,
-            name: 'James Bow',
-            role: 'FE',
-        },
-        {
-            key: 5,
-            name: 'Jane Dough',
-            role: 'BE',
-        },
-    ]
+    if (loading) return loadingScreenShow()
+    else if (error) return errorNotify(`Error! ${error}`)
```

Comment:
No need to put else here since your already returning in your first if-statement
```suggestion
    if (error) return errorNotify(`Error! ${error}`)
```